swatch-producer-aws/deploy/clowdapp.yaml

```diff
@@ -44,9 +44,11 @@ parameters:
   - name: SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT
     value: http://swatch-subscription-sync-service:8000/api/rhsm-subscriptions/v1
   - name: AWS_MARKETPLACE_ENDPOINT_URL
-    value: 'http://localhost:8101/aws-marketplace/'
+    value: 'http://moto:5000/'
   - name: AWS_MANUAL_SUBMISSION_ENABLED
-    value: 'false'
+    value: 'true'
+  - name: AWS_MARKETPLACE_ENDPOINT_OVERRIDE
+    value: 'true'
```

Comment:
Instead of modifying the defaults on the template, I'd rather make these changes in app-interface on the ephemeral definition only.